httphandler/build.py

```diff
@@ -6,60 +6,63 @@
 
 BASE_GETTER_CONST = "github.com/kubescape/kubescape/v2/core/cautils/getter"
 
-def checkStatus(status, msg):
+def check_status(status, msg):
     if status != 0:
         sys.stderr.write(msg)
         exit(status)
 
 
-def getBuildDir():
-    currentPlatform = platform.system()
-    buildDir = "build/"
+def get_build_dir():
+    current_platform = platform.system()
+    build_dir = "build/"
 
-    if currentPlatform == "Windows": return os.path.join(buildDir, "windows-latest") 
-    if currentPlatform == "Linux": return os.path.join(buildDir, "ubuntu-latest")  
-    if currentPlatform == "Darwin": return os.path.join(buildDir, "macos-latest")  
-    raise OSError("Platform %s is not supported!" % (currentPlatform))
+    if current_platform == "Windows": build_dir += "windows-latest"
```

Comment:
I believe that previous code was more platform generic when using `os.path.join`; As now the build_dir has a hardcoded path separator (which is incorrect for Windows)